build/pkgs/ecl/SPKG.rst

```diff
@@ -43,7 +43,7 @@ Special Update/Build Instructions
 
 -  Do NOT quote SAGE_LOCAL when setting CPPFLAGS and/or LDFLAGS,
    in spkg-install as this caused the build to break. See
-   http://trac.sagemath.org/sage_trac/ticket/10187#comment:117
+   :issue:`10187`#comment:117
```

Comment:
Pretty sure the # has to be inside the backticks

Reply:
Thanks